### PR TITLE
Add DotVVM.Framework.Testing project

### PR DESCRIPTION
**Pull request by exyi**
The project is meant to contain helper methods for testing dotvvm controls

### CODE REVIEW

File: src/Tools/CI/restore.cmd [hunk 1]
```diff
@@ -21,6 +21,9 @@ dotnet restore
 cd ../DotVVM.Framework.PerfTests
 dotnet restore
 
+cd ../DotVVM.Framework.Testing
```
Comment (quigamdev) [inline src/Tools/CI/restore.cmd:24]: This file actually is not needed. We should remove it.

File: src/Tools/CI/release.cmd [hunk 2]
```diff
@@ -11,6 +11,9 @@ dotnet build ../../DotVVM.Framework -c Release
 dotnet restore ../../DotVVM.Framework.Hosting.AspNetCore
 dotnet build ../../DotVVM.Framework.Hosting.AspNetCore -c Release
 
+dotnet restore ../../DotVVM.Framework.Testing
+dotnet build ../../DotVVM.Framework.Testing -c Release
```
Comment (quigamdev) [inline src/Tools/CI/release.cmd:15]: This file actually is not needed. We should remove it.

File: src/Tools/CI/build.cmd [hunk 3]
```diff
@@ -11,6 +11,9 @@ dotnet build ../../DotVVM.Framework
 dotnet restore ../../DotVVM.Framework.Hosting.AspNetCore
 dotnet build ../../DotVVM.Framework.Hosting.AspNetCore
 
+dotnet restore ../../DotVVM.Framework.Testing
```
Comment (quigamdev) [inline src/Tools/CI/build.cmd:14]: This file actually is not needed. We should remove it.

**Review by quigamdev — APPROVED** [review]

File: src/Tools/CI/build.cmd (hunk 3 above)
Comment (exyi) [reply src/Tools/CI/build.cmd:14]: OK, won't do that in this PR though. Could you please do that when you have a bit of time?

**Pull request merged**